static/assets/js/entity-reference/app.1.2.js

```diff
@@ -20,13 +20,13 @@
 
     function EntityReferenceCtrl($window, $http, $sce, $timeout, bbWait, localStorageService) {
         var self = this;
-        this.apiTitle = '';
         this.showErrorMessage = false;
+        this.swaggerCacheName = 'swaggerResponseCache-' + this.apiTitle;
 
         this.$onInit = onInit;
 
         function onInit() {
-            var swaggerResponseCache = localStorageService.get('swaggerResponseCache');
+            var swaggerResponseCache = localStorageService.get(self.swaggerCacheName);
```

Comment:
Might want to investigate:

- Valid Characters for setItem(...) key & if that matches what might be passed to apiTitle (i.e. unicode? can html be passed to apiTitle?)
- Is there a size limit to local storage or browser restrictions that might need to be addressed? (probably unrelated to this change, but something to think about as we think about multiple teams using this)